versioned/tiered/rocksdb/src/main/java/org/projectnessie/versioned/rocksdb/RocksFragment.java

```diff
@@ -32,19 +33,30 @@
 /**
  * A RocksDB specific implementation of {@link org.projectnessie.versioned.tiered.Fragment} providing
  * SerDe and Condition evaluation.
+ *
+ * <p>Conceptually, this is matching the following JSON structure:</p>
+ * <pre>{
+ *   "id": &lt;ByteString&gt;, // ID
+ *   "dt": &lt;int64&gt;,      // DATETIME
+ *   "keys": [                 // KEY_LIST
+ *     [
+ *       &lt;String&gt;
+ *     ]
+ *   ]
+ * }</pre>
  */
 class RocksFragment extends RocksBaseValue<Fragment> implements Fragment {
   static final String KEY_LIST = "keys";
 
-  private final ValueProtos.Fragment.Builder builder = ValueProtos.Fragment.newBuilder();
+  private final ValueProtos.Fragment.Builder fragmentBuilder = ValueProtos.Fragment.newBuilder();
```

Comment:
Why the rename?

Reply:
In my opinion, builder is an overly generic name. This gets problematic if we have a scope that also has another type of **Builder**, such as a **Key.Builder**.
Is there a reason to keep the generic name?